source/elements/oneMKL/source/domains/spblas/operations/spmm.rst

```diff
@@ -302,7 +302,9 @@ spmm
 
    A_view
       Specifies which part of the handle should be read as described by
-      :ref:`onemkl_sparse_matrix_view`.
+      :ref:`onemkl_sparse_matrix_view`. The ``type_view`` field must be
+      ``matrix_descr::general`` and the ``uplo_view`` and ``diag_view`` fields
+      are ignored.
```

Comment:
Is it useful to generate an error instead of ignoring?

Reply:
We can't generate an error with the current API. The `uplo_view` and `diag_view` don't have anything like a `none` value. We would need to use `std::optional` in the matrix view but I think it would make the type a bit clunky.